docs/wiki/Fetching latest changes - Scenario 1 Module library only.md

```diff
@@ -0,0 +1,149 @@
+In this scenario you have onboarded only the module library as described in [Getting started Scenario 1- Consume library](./Getting%20started%20-%20Scenario%201%20Consume%20library) and therefore you fetch lastest changes for the modules.
+
+Modules can be stored in an accessible location like local, Template Specs, Bicep Registry or ADO universal packages.
+
+The update process is the following:
+
+### _Navigation_
+
+- [1. Backup your local copy of the library](#1-backup-your-local-copy-of-the-library)
+- [2. Download the library](#2-download-the-library)
+- [3. (Optional) Convert library to ARM](#3-optional-convert-library-to-arm)
+- [4. (Optional) Customize modules](#4-optional-customize-modules)
+- [5. Test and publish modules](#5-test-and-publish-modules)
+# 1. Backup your local copy of the library
+
+<details>
+<summary>Rename your local repository</summary>
+
+Assuming the local repository location is `'D:\ResourcesModules'` rename it in `'D:\ResourcesModules_Backup'`.
+
+The backup folder can be used to compare customized modules with the ones coming from the latest version and implement the required changes.
+
+This can be done, for example, by the `'Compare selected'` [function](https://vscode.one/diff-vscode/) of Visual Studio Code
+
+</details>
```

Comment:
```suggestion
Rename your local repository. Assuming the local repository location is `'D:\ResourcesModules'` rename it in `'D:\ResourcesModules_Backup'`.

The backup folder can be used to compare customized modules with the ones coming from the latest version and implement the required changes.

This can be done, for example, by the `'Compare selected'` [function](https://vscode.one/diff-vscode/) of Visual Studio Code

```